common/app/services/newsletters/NewsletterApi.scala

```diff
@@ -44,73 +44,51 @@ object GroupedNewsletterResponse {
 
 // TODO: Find a better way to define this that means Frontend doesn't have knowledge of the fields returned.
```

Comment:
Would be interested to hear if you have any thoughts about how this 'TODO' could be addressed (only out of pure curiosity!)

Reply:
We are thinking of replacing the call to grouped newsletters to a call to newsletters and build the groups on the frontend. Similarly to what is done for /user-prefs